views/about_me.erb

```diff
@@ -0,0 +1,28 @@
+<html>
```

Comment:
Watch out for indentation, once you open a tag and start nesting, each subsequent tag should be indented